src/main/java/com/maxmind/minfraud/response/IpRiskReason.java

```diff
@@ -10,6 +10,11 @@ public final class IpRiskReason extends AbstractModel {
     private final String code;
     private final String reason;
 
+    /**
```

Comment:
Given that you added it elsewhere, maybe we should have a description here and below.